apps/src/code-studio/components/AssetManager.jsx

```diff
@@ -49,7 +52,11 @@ var AssetManager = React.createClass({
    * @param xhr
    */
   onAssetListReceived: function (xhr) {
-    assetListStore.reset(JSON.parse(xhr.responseText));
+    let parsedResponse = JSON.parse(xhr.responseText);
```

Comment:
Will solve this within the new API per comment above. Note we have many other places in the project system code where we need to clean up how we handled failed calls to `JSON.parse()` - to be solved another day...